Marlin/Configuration.h

```diff
@@ -387,7 +387,9 @@ const bool Z_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the logic
 #define min_software_endstops true // If true, axis won't move to coordinates less than HOME_POS.
 #define max_software_endstops true  // If true, axis won't move to coordinates greater than the defined lengths below.
 
-// @section machine
```

Comment:
Probably good to keep this section directive, even though the configurator is currently stale.